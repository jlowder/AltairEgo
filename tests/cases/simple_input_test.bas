10 INPUT A,B
20 PRINT A,B
30 END
