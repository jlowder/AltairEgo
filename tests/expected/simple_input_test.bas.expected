Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
?  0             0
OK
